<!DOCTYPE html>
<html lang="en">
  <head>
    <base href="/">
    <title>Gingras lab</title>
    <meta name="Description" content="The laboratory of Dr Anne-Claude Gingras at the Lunenfeld in Toronto.">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <meta name="robots" content="noindex,nofollow">
    
    <link rel="apple-touch-icon" sizes="180x180" href="assets/favicon/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="assets/favicon/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="assets/favicon/favicon-16x16.png">
    <link rel="manifest" href="site.webmanifest">
    <link rel="mask-icon" href="assets/favicon/safari-pinned-tab.svg" color="#5bbad5">
    <meta name="theme-color" content="#f64f59">

    <link href="https://fonts.googleapis.com/css?family=Montserrat|Lato&display=swap" rel="stylesheet">
    <link href="css/style.css" rel="stylesheet">
    <link href="css/news.css" rel="stylesheet">
    <script src="bundle.js" defer></script>
  </head>
  <body>
    <nav class="navbar">
      <span class="navbar__left">
        <a class="navbar__home" href="./">Gingras lab</a>
        <span class="navbar__links">
          <a href="research/">Research</a>
          <a href="resources/">Resources</a>
          <a href="publications/">Publications</a>
          <a href="people/">People</a>
          <a href="news/">News & Openings</a>
        </span>
      </span>
      <span class="navbar__right">
        <a href="contact/">
          <svg
            xmlns="http://www.w3.org/2000/svg"
            aria-labelledby="envelopeIcon"
            height="30"
            role="img"
            viewBox="0 0 448 512"
          >
            <title id="envelopeIcon">Envelope</title>
            <path d="M224 320c-26.72.13-50.54-21.67-64.7-32C97.33 243 78 228.57 64 217.77V360a24 24 0 0 0 24 24h272a24 24 0 0 0 24-24V217.77c-14 10.8-33.33 25.24-95.28 70.2C275 298 250.88 320.13 224 320zm136-192H88a24 24 0 0 0-24 24v25.17c24.35 19 23.43 19.12 114.12 84.93 10.51 7.67 31.38 26.12 45.88 25.9 14.49.22 35.38-18.24 45.88-25.9 90.69-65.81 89.75-66 114.12-84.93V152a24 24 0 0 0-24-24z" class="fa-primary" />
          </svg>
        </a>
        <span class="navbar__menu-container">
          <button
            class="navbar__menu-button"
            type="button"
          >
            <svg
              xmlns="http://www.w3.org/2000/svg"
              aria-labelledby="menuIcon"
              height="25"
              role="img"
              viewBox="0 0 448 512"
            >
              <title id="menuIcon">Menu</title>
              <path d="M16 132h416c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H16C7.163 60 0 67.163 0 76v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h416c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H16c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"/>
            </svg>
          </button>
          <ul class="navbar__menu">
            <li><a class="nav-link" href="research/">Research</a></li>
            <li><a class="nav-link" href="resources/">Resources</a></li>
            <li><a class="nav-link" href="publications/">Publications</a></li>
            <li><a class="nav-link" href="people/">People</a></li>
            <li><a class="nav-link" href="news/">News & Openings</a></li>
            <li><a class="nav-link" href="contact/">Contact</a></li>
          </ul>
        </span>
      </span>
    </nav>
    <main class="main news">
      <section>
        <header>
          <h1>
            Open positions
          </h1>
          <img
            alt="Magnifying glass searching for a person"
            height="25"
            src="assets/images/icons/user-search.svg"
          />
        </header>
        <h2>Active listings</h2>
        <ul>
          <li>
            Postdoctoral position in chemical proteomics (<a href="https://apply.interfolio.com/66200">application</a>).
          </li>
          <li>
            Postdoctoral position in computational proteomics (<a href="https://apply.interfolio.com/58107">application</a>).
          </li>
          <li>
            Research Associate position in chemical proteomics
            (<a href="https://contact2.mshri.on.ca/hr/DEFAULT.ASP?page=jobDetail&amp;job=1225">application</a>).
          </li>
        </ul>
        <h2>Postdocs</h2>
        <p>
          The laboratory is always interested in recruiting highly motivated postdoctoral fellows with a strong
          publication record in signal transduction, cell biology, biochemistry, bioinformatics and/or proteomics.
          Please forward your CV, references and research interests to
          <a class="nav-link" href="mailto:gingras@lunenfeld.ca">Dr. Anne-Claude Gingras</a>.
        </p>
        <h2>Graduate students</h2>
        <p>
          The Gingras lab is part of the
          <a href="http://www.moleculargenetics.utoronto.ca/">Department of Molecular Genetics</a>
          at the
          <a href="https://www.utoronto.ca/">University of Toronto</a>,
          which has a central admission committee and a rotation system. Graduate students interested in doing a
          PhD in the laboratory must first be accepted in the Department of Molecular Genetics</a>.
        </p>
        <h2>Summer students</h2>
        <p>
          Summer students are exclusively selected from successful applicants to the
          <a href="http://research.lunenfeld.ca/rtc/">Research Training Center (RTC)</a>
          at the Lunenfeld-Tanenbaum Research Institute.
          <a href="http://research.lunenfeld.ca/rtc/?page=Summer%20Research%20Program">Applications</a>
          are available online and need to be filled by February 15th yearly.
        </p>
      </section>
      <section class="news__articles">
        <header>
          <h1>
            News
          </h1>
          <img
            alt="Newspaper"
            height="25"
            src="assets/images/icons/newspaper.svg"
          />
        </header>
        <div class="news__years">
          <a href="news/#2019">2019</a>
          <a href="news/#2018">2018</a>
          <a href="news/#2017">2017</a>
          <a href="news/#2016">2016</a>
          <a href="news/#2015">2015</a>
          <a href="news/#2014">2014</a>
          <a href="news/#2013">2013</a>
          <a href="news/#2012">2012</a>
          <a href="news/#2011">2011</a>
          <a href="news/#2010">2010</a>
        </div>
        <h2 id="2019">2019</h2>
        <ul class="news__list">
          <li>
            <time>July 31, 2019</time>
            <h3>Postdoctoral and research associate positions</h3>
            <p>
              Postdoctoral positions in <a href="https://apply.interfolio.com/58107">computational proteomics</a> 
              (joint with Hannes Roest at UofT) and <a href="https://apply.interfolio.com/66200">chemical proteomics</a>
              are available in the Gingras lab. A Research Associate position is also available in
              <a href="https://contact2.mshri.on.ca/hr/DEFAULT.ASP?page=jobDetail&amp;job=1225">chemical proteomics</a>.
            </p>
          </li>
          <li>
            <time>June 19, 2019</time>
            <h3>GIX: Gene Information eXtension published</h3>
            <p>
              James' new Chrome and Firefox extension to quickly annotate a gene product directly in
              a web browser is now published at <a href="https://www.nature.com/articles/s41592-019-0477-9">Nature Methods</a>.
              You can download it from the <a href="https://gene-info.org">GIX</a> homepage.
            </p>
          </li>
          <li>
            <time>February 7, 2019</time>
            <h3>Rewiring the BET interactome</h3>
            <p>
              See our manuscript by
              <a href="https://www.sciencedirect.com/science/article/pii/S1097276518309481?via%3Dihub">Lambert, Picaud et al.</a>
              that reports the effects of the drug JQ1 on the interactome of the BET family of bromodomains online at Molecular
              Cell. A great collaboration between the Gingras and Filippakopoulos labs, with help from many colleagues across the world.
            </p>
          </li>
        </ul>
        <h2 id="2018">2018</h2>
        <ul class="news__list">
          <li>
            <time>November 29, 2018</time>
            <h3>Computational postdoc positions available</h3>
            <p>
              The Gingras and
              <a href="http://www.roestlab.org">Rost</a>
              labs are looking for postdoctoral fellows in Computational Proteomics to lead the development
              of the next generation of tools for Data Independent Analysis
              (<a href="https://apply.interfolio.com/30295/positions">application</a>).
            </p>
          </li>
          <li>
            <time>August 14, 2018</time>
            <h3>Become my newest colleague!</h3>
            <p>
              The <a href="http://lunenfeld.ca"">Lunenfeld</a> is recruiting 3 new PIs in the Fall. The Molecular,
              Cell and Systems Biology position is now open - cross-appointment at the University of Toronto 
              (<a href="https://apply.interfolio.com/53041">application)</a>. See all
              <a href="https://apply.interfolio.com/30295/positions">open positions</a>.
            </p>
          </li>
          <li>
            <time>July 11, 2018</time>
            <h3>Lentiviral toolbox: BioID across cell types!</h3>
            <p>
              Payman Samavarchi-Tehrani and colleagues report in
              <a href="http://www.mcponline.org/content/early/2018/07/10/mcp.TIR118.000902.abstract">Molecular and Cellular Proteomics</a>
              a collection of lentiviral vectors to enable proximity-dependent biotinylation across cell types.
            </p>
          </li>
          <li>
            <time>March 15, 2018</time>
            <h3>Congrats Boris, recipient of NSERC studentship</h3>
            <p>
              PhD student Boris Dyakov recently received an award from the
              <a href="http://www.nserc-crsng.gc.ca/index_eng.asp">Natural Sciences and Engineering Research Council of Canada</a>
              for his work aiming at characterizing nuclear bodies using proteomic tools.
            </p>
          </li>
          <li>
            <time>February 1, 2018</time>
            <h3>Cytosolic RNA granules and bodies out in Mol Cell</h3>
            <p>
              Congratulations to all co-authors and to PDF Jiyoung Youn, profiled at
              <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(17)30977-2">Molecular Cell</a>
              for her paper selected as a
              <a href="http://www.cell.com/molecular-cell/meet-the-author/ji-young-youn">Featured Article</a>
              (free). Also see the nice
              <a href="http://www.cell.com/molecular-cell/fulltext/S1097-2765(18)30049-2">Preview</a>
              by Simon Alberti.
            </p>
          </li>
        </ul>
        <h2 id="2017">2017</h2>
        <ul class="news__list">
          <li>
            <time>December 18, 2017</time>
            <h3>We are growing! Welcome to our new students!</h3>
            <p>
              The rotation students were great this year: very happy to recruit Kento Abe, Julia Kitaygorodsky,
              Shimon Rosenthal (co-supervised with Ian Scott) and Francesco Zangari to the lab.
            </p>
          </li>
          <li>
            <time>June 29, 2017</time>
            <h3>ProHits-Viz paper is out at Nature Methods</h3>
            <p>
              Knight et al. publishes web-tool to improve visualization of protein interaction data
              (<a href="https://www.nature.com/articles/nmeth.4330.epdf?author_access_token=Mxscfd4TsFnZDOMRVLu6q9RgN0jAjWel9jnR3ZoTv0N_r0kXKLsUVDr0iDmCOCg724OKwf_HwmR4LS6tUTcLnv0lZ95VTwz0lijrGL0nG0hlWQsNj9aa71B1ICe2lxjt">read-only version</a>).
            </p>
          </li>
          <li>
            <time>May 29, 2017</time>
            <h3>Congrats to our students!</h3>
            <p>
              Chris Go and Jonathan Roth are each awarded an Ontario Graduate Scholarship while Benjamin
              Piette receives a FRQNT from the Quebec government.
            </p>
          </li>
          <li>
            <time>April 4, 2017</time>
            <h3>Papers delineate MOB1 role in Hippo signaling</h3>
            <p>
              Amber Couzens, Shawn Xiong and collaborators used proteomics and structural and biophysical
              studies to uncover the function of MOB1 in the regulation of the Hippo signaling pathway.
              See both papers at Mol Cell Proteomics.
            </p>
          </li>
        </ul>
        <h2 id="2016">2016</h2>
        <ul class="news__list">
          <li>
            <time>November 22, 2016</time>
            <h3>Fresh off the press: phosphatases in mitosis</h3>
            <p>
              St-Denis et al. report in this issue of
              <a href="http://www.cell.com/cell-reports/abstract/S2211-1247(16)31513-3">Cell Reports</a>
              the identification of phosphatases
              implicated in mitosis, and the elucidation of their interactomes. 
            </p>
          </li>
          <li>
            <time>September 20, 2016</time>
            <h3>Congratulations to Ben, Boris and Cassandra!</h3>
            <p>
              Our new students get departmental awards: a Cecil Yip doctoral award from the Donnelly Centre to
              Ben, the Norman Bethune Award to Boris and the Hannah Farkas-Himsley and Alexander Himsley ‎Memorial
              Prize to Cassandra (both from Molecular Genetics). Cassandra and Boris each also received OSOTF
              funding from the Lunenfeld-Tanenbaum Research Institute.
            </p>
          </li>
          <li>
            <time>September 20, 2016</time>
            <h3>New TFRI grant to "Team Hippo"</h3>
            <p>
              A team composed of Jeff Wrana, Lil Attisano, Frank Sicheri, Helen McNeill, Steve Gallinger and
              Anne-Claude Gingras receives 2.25M to target the Hippo tumor suppressor pathway in cancer. Thanks to
              TFRI, donors and volunteers!
            </p>
          </li>
          <li>
            <time>June 1, 2016</time>
            <h3>Welcome Shen, new PDF in the group</h3>
            <p>
              Shen Zhang recently joined from the Dalian Institute of Chemical Physics, Chinese Academy of
              Sciences and brings new expertise in quantitative proteomics.
            </p>
          </li>
          <li>
            <time>May 2, 2016</time>
            <h3>ProHits 4.0 released and published</h3>
            <p>
              The ProHits 4.0 publication in press at J Proteomics (Liu et al) describes the use of our
              <a href="http://prohitsms.com/">ProHits LIMS</a>
              for Data Independent Acquisition through the incorporation of
              <a href="http://diaumpire.sourceforge.net/">DIA-Umpire</a>,
              <a href="http://proteomics.ucsd.edu/software-tools/msplit-dia/">MSPLIT-DIA</a>
              and
              <a href="http://mapdia.sourceforge.net/Main.html">mapDIA</a>.
            </p>
          </li>
          <li>
            <time>May 1, 2016</time>
            <h3>Welcome Bhavisha and Debbie</h3>
            <p>
              Bhavisha is now assisting Dushyandi in the essential lab manager position while Debbie is working
              on our recently funded CCSRI Innovation grant. Amber is back to continue her work on Hippo signaling.
            </p>
          </li>
          <li>
            <time>March 12, 2016</time>
            <h3>Welcome Stefan, farewell Wade</h3>
            <p>
              Welcome to Stefan Maier, PDF who will be developing new proteomics approaches. We are sad to say
              goodbye to our longtime member Wade Dunham who is starting a new venture outside of the city.
            </p>
          </li>
          <li>
            <time>February 5, 2016</time>
            <h3>Thank you, Canadian Cancer Society!</h3>
            <p>
              Thanks to the donors and volunteers of the Canadian Cancer Society, our lab received a two-year
              grant to investigate endocytic organelles as sites of cellular growth control.
            </p>
          </li>
          <li>
            <time>January 1, 2016</time>
            <h3>Happy 2016 and welcome all new members!</h3>
            <p>
              Welcome to the lab to our new students Ben, Boris and Cassandra! Dushyandi replaces Marilyn as
              co-manager of the lab. Good luck to Marilyn and hubby Prof Smith in Montreal!
            </p>
          </li>
        </ul>
        <h2 id="2015">2015</h2>
        <ul class="news__list">
          <li>
            <time>November 10, 2015</time>
            <h3>MSPLIT-DIA paper is out in Nature Methods</h3>
            <p>
              In a collaboration with
              <a href="http://proteomics.ucsd.edu/">Nuno Bandeira, UCSD</a>,
              we are reporting in
              <a href="https://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3655.html">Nature Methods</a>
              a new spectral matching approach for peptide identification from DIA. This will be incorporated in
              the next
              <a href="http://prohitsms.com/">ProHits</a>
              release.
            </p>
          </li>
          <li>
            <time>August 7, 2015</time>
            <h3>Thank you, Cancer Research Society!</h3>
            <p>
              Our lab receives 2 year grant from the Cancer Research Society. We thank the supporters and
              volunteers of the CRS for all their efforts!
            </p>
          </li>
          <li>
            <time>July 30, 2015</time>
            <h3>JP Lambert awarded CRS career transition award</h3>
            <p>
              Jean-Philippe is awarded a prestigious Scholarship for the Next Generation of Scientists from the
              Cancer Research Society!
            </p>
          </li>
          <li>
            <time>July 2, 2015</time>
            <h3>Association of E4orf4 with two major phosphatases</h3>
            <p>
              In collaboration with the Branton lab (McGill), MSc student Yiwang Zhou uncovers new mechanisms of
              signaling regulation by Adenovirus E4orf4: E4orf4 regulates Hippo signaling though PP1 and PP2A.
            </p>
          </li>
          <li>
            <time>June 18, 2015</time>
            <h3>A Parkinson Society of Canada fellowship to Geoff!</h3>
            <p>
              Geoff Hesketh, PDF, receives the prestigious award for his mass spectrometry based characterization
              of trafficking proteins implicated in Parkinson disease pathogenesis.
            </p>
          </li>
          <li>
            <time>March 31, 2015</time>
            <h3>Network Biology Collaborative Centre is a GIN node</h3>
            <p>
              New Funding from Genome Canada for the SMART and proteomics facilities at LTRI. The
              <a href="http://nbcc.lunenfeld.ca/">NBCC</a>
              team led by Jeff Wrana and Anne-Claude Gingras becomes a Genomics Innovation network node.
            </p>
          </li>
          <li>
            <time>March 25, 2015</time>
            <h3>Worm CCM3 regulates excretory canal</h3>
            <p>
              Ben Lant publishes new research in
              <a href="http://www.ncbi.nlm.nih.gov/pubmed/25743393">Nature Communications</a>;
              work funded by joint CIHR to Drs. Derry and Gingras.
            </p>
          </li>
          <li>
            <time>February 7, 2015</time>
            <h3>Non-enzymatic role for myotubularins in cell cycle</h3>
            <p>
              Nicole St-Denis et al. report that MTMR3 & 4 control abscission via interactions with PLK1 and CEP55.
            </p>
          </li>
          <li>
            <time>January 30, 2015</time>
            <h3>DIA-Umpire: a new way to look at SWATH data</h3>
            <p>
              A collaboration between the Gingras and Nesvizhskii labs leads to the development of a complete
              informatics pipeline for the analysis of DIA/SWATH data (publication by Tsou et al. in Nature Methods).
            </p>
          </li>
        </ul>
        <h2 id="2014">2014</h2>
        <ul class="news__list">
          <li>
            <time>December 16, 2014</time>
            <h3>Lap-Chee Tsui Publication award to Amber Couzens</h3>
            <p>
              Read again the winning publication on the
              <a href="http://stke.sciencemag.org/content/6/302/rs15.long">Hippo interactome</a>.
            </p>
          </li>
          <li>
            <time>November 28, 2014</time>
            <h3>Improving AP-MS data visualization</h3>
            <p>
              James Knight and colleagues publish a
              <a href="http://prohitstools.mshri.on.ca/">web tool</a>
              for visualization of interaction proteomics data.
            </p>
          </li>
          <li>
            <time>September 15, 2014</time>
            <h3>How to score true interactions in BioID experiment</h3>
            <p>
              <a href="https://www.sciencedirect.com/science/article/pii/S1874391914004357?via%3Dihub">JP Lambert et al.</a>
              thoroughly analyze background subtraction in BioID and compare BioID to standard AP-MS for
              chromatin proteins.
            </p>
          </li>
          <li>
            <time>July 17, 2014</time>
            <h3>Cell paper by the Gingras and Lindquist labs</h3>
            <p>
              <a href="https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(14)00740-5">Taipale et al.</a>
              report a quantitative chaperone interaction network that reveals the architecture of cellular protein
              homeostasis pathways.
            </p>
          </li>
          <li>
            <time>April 15, 2014</time>
            <h3>Chris Go receives doctoral award</h3>
            <p>
              Congrats to Chris Go on getting a competitive Frederick Banting and Charles Best Canada Graduate
              Scholarship.
            </p>
          </li>
        </ul>
        <h2 id="2013">2013</h2>
        <ul class="news__list">
          <li>
            <time>November 20, 2013</time>
            <h3>Stopping the Hippos</h3>
            <p>
              Hundreds of new connections and mechanisms of phospho-recognition in the Hippo pathway published by
              <a href="https://stke.sciencemag.org/content/6/302/rs15.long">Amber Couzens et al.</a>
              in Science Signaling - also see accompanying
              <a href="https://stke.sciencemag.org/content/6/302/pe36">perspective</a>
              by Eric Weiss.
            </p>
          </li>
          <li>
            <time>October 19, 2013</time>
            <h3>AP-SWATH paper in Nature Methods</h3>
            <p>
              In a collaboration with
              <a href="https://sciex.com/">AB SCIEX</a>
              and the support of
              <a href="http://support.supportsinai.com/site/PageServer?pagename=mainsite_getinvolved_venturesinai">Venture Sinai</a>,
              <a href="https://www.nature.com/articles/nmeth.2702">Lambert et al.</a>
              present AP-SWATH which rapidly maps differential interactomes.
            </p>
          </li>
          <li>
            <time>June 16, 2013</time>
            <h3>Congrats Nicole and Chris</h3>
            <p>
              Postdoc Nicole St-Denis and grad student Chris Go each obtained competitive CIHR support for
              their studies of phosphatases.
            </p>
          </li>
          <li>
            <time>June 16, 2013</time>
            <h3>Publication of the CRAPome</h3>
            <p>
              The Nesvizhskii and Gingras teams led an international effort to create a repository to help filter
              the cra... hmm... contaminants out of AP-MS data. See the paper by
              <a href="https://www.nature.com/articles/nmeth.2557">Datta Mellacheruvu et al.</a>,
              in Nature Methods.
            </p>
          </li>
          <li>
            <time>May 1, 2013</time>
            <h3>Funding from Genome Canada</h3>
            <p>
              Anne-Claude Gingras and Mike Tyers (IRIC, Montreal) co-lead an international team for the development of
              ProHits Next Generation, a platform for interaction proteomics. Funding from Genome Canada was
              awarded in the recent Bioinformatics and Computational Biology competition.
            </p>
          </li>
          <li>
            <time>May 1, 2013</time>
            <h3>Funding from NSERC awarded</h3>
            <p>
              We recently obtained a NSERC Discovery grant and an Accelerator Supplement to study the specificity
              in RNA storage and degradation associated to p bodies and stress granules.
            </p>
          </li>
        </ul>
        <h2 id="2012">2012</h2>
        <ul class="news__list">
          <li>
            <time>August 1, 2012</time>
            <h3>Special issue of Proteomics</h3>
            <p>
              Have a look at the special issue of Proteomics on
              <a href="https://onlinelibrary.wiley.com/toc/16159861/2012/12/10">Protein Complexes and Interaction Networks</a>
              co-Edited by Alexey Nesvizhskii and Anne-Claude Gingras. 
            </p>
          </li>
          <li>
            <time>June 1, 2012</time>
            <h3>James received Heart and Stroke funding!</h3>
            <p>
              Congratulations to James on receiving a prestigious fellowship from the Heart and Stroke
              Foundation of Canada!
            </p>
          </li>
        </ul>
        <h2 id="2011">2011</h2>
        <ul class="news__list">
          <li>
            <time>December 1, 2011</time>
            <h3>Nicole receives TD Bank scholarship</h3>
            <p>
              Postdoctoral fellow Nicole St-Denis is awarded prestigious TD Bank postdoctoral fellowship for
              her studies of phosphatases in mitosis.
            </p>
          </li>
          <li>
            <time>September 12, 2011</time>
            <h3>ProHits version 2.0.0</h3>
            <p>
              Major new ProHits release - SAINT can be run seamlessly.
            </p>
          </li>
          <li>
            <time>August 19, 2011</time>
            <h3>Profile of Anne-Claude in ASBMB Today</h3>
            <p>
              Zick Zagorski pens the
              <a href="http://www.asbmb.org/asbmbtoday/201108/sciencefocus/Gingras/">profile of Anne-Claude Gingras</a>
              in this month's issue of ASBMB Today.
            </p>
          </li>
          <li>
            <time>August 19, 2011</time>
            <h3>New releases for SAINT and ProHits</h3>
            <p>
              We keep updating our software. Check our
              <a class="nav-link" href="resources/">Resources</a>
              section for the latest downloads of our interaction proteomics tools.
            </p>
          </li>
          <li>
            <time>June 27, 2011</time>
            <h3>Publications by two of our students</h3>
            <p>
              Graduate students Michelle Kean and Wade Dunham each got their first publication from the
              Gingras lab. Congrats to both! Access the publications at
              <a href="http://www.jbc.org/content/early/2011/05/11/jbc.M110.214486.abstract">J Biol Chem</a>
              and
              <a href="https://onlinelibrary.wiley.com/doi/10.1002/pmic.201000571/abstract">Proteomics</a>.
            </p>
          </li>
        </ul>
        <h2 id="2010">2010</h2>
        <ul class="news__list">
          <li>
            <time>December 10, 2010</time>
            <h3>SAINT paper published</h3>
            <p>
              The Gingras and Nesvizhskii labs publish in Nature Methods the new version of the statistical
              tool SAINT for scoring interaction proteomics data. Download the free software at
              <a href="http://saint-apms.sourceforge.net/Main.html">sourceforge</a>.
            </p>
          </li>
          <li>
            <time>October 13, 2010</time>
            <h3>Just published!</h3>
            <p>
              <a href="https://www.nature.com/nbt/journal/v28/n10/full/nbt1010-1015.html">ProHits: integrated software for mass spectrometry-based interaction proteomics</a>.
              The Gingras and Tyers labs publish in Nature Biotechnology
              their interaction proteomics data management system ProHits. Download the free software at
              <a href="http://prohitsms.com">prohitsms.com</a>.
            </p>
          </li>
          <li>
            <time>May 21, 2010</time>
            <h3>Fresh off the press</h3>
            <p>
              <a href="https://www.sciencemag.org/cgi/content/abstract/328/5981/1043">A global protein kinase and phosphatase interaction network in yeast</a>.
              The Tyers, Nesvizhskii and Gingras labs team up to provide a comprehensive interaction map for
              yeast kinases and phosphatases (Breitkreutz et al., Science, 2010).
            </p>
          </li>
          <li>
            <time>April 29, 2010</time>
            <h3>New funding announced</h3>
            <p>
              Dr. Gingras is part of a research team led by Prof. Pawson that was awarded a $7.7 million GL2 grant
              from Ontario's Ministry of Research and Innovation, which the team plans to use for understanding the
              assembly and function of dynamic signaling networks in complex diseases.
            </p>
          </li>
        </ul>
      </section>
    </main>
    <footer class="footer-nav">
      <div class="footer__links">
        <a class="nav-link" href="research/">Home</a>
        <a class="nav-link" href="research/">Research</a>
        <a class="nav-link" href="resources/">Resources</a>
        <a class="nav-link" href="publications/">Publications</a>
        <a class="nav-link" href="people/">People</a>
        <a class="nav-link" href="news/">News & Openings</a>
        <a class="nav-link" href="contact/">Contact</a>
      </div>
      <small>© 2019 Gingras lab.</small>
    </footer>
  </body>
</html>
